tests/test_tag_object.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+try:
+    from unittest.mock import MagicMock, patch, call
+except ImportError:
+    from mock import MagicMock, patch, call
+from datadog_lambda.tag_object import tag_object
+
+
+class TestTagObject(unittest.TestCase):
+    payload = {
+        "hello": "world",
+        "anotherThing": {"foo": "bar", "blah": None, "nice": True},
+        "vals": [{"thingOne": 1}, {"thingTwo": 2}],
```

Comment:
I would suggest adding test coverage for redaction here

Reply:
Good call

Reply:
Done